llama-index-integrations/tools/llama-index-tools-cassandra/pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.mypy]
+disallow_untyped_defs = true
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+authors = ["Patrick McFadin <patrick@datastax.com>"]
+description = "llama-index tools Apache Cassandra® integration"
+license = "MIT"
+name = "llama-index-tools-cassandra"
+packages = [{include = "llama_index/"}]
+readme = "README.md"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+llama-index-core = "^0.10.0"
```

Comment:
We should add Cassandra as a dependency here

Reply:
Oops. Good catch. Added dependency for Cassio ^0.1.7